packages/devtools_app/lib/src/inspector/flutter/layout_tab.dart

```diff
@@ -0,0 +1,190 @@
+  FlexProperties properties;
+
+  @override
+  Widget build(BuildContext context) {
+    final List<Widget> children = [
+      for (RemoteDiagnosticsNode child in node.diagnostic.childrenNow)
+        Expanded(
+          flex: 1,
+          child: Container(
+            decoration: BoxDecoration(
+              color: Theme
+                .of(context)
+                .backgroundColor,
+              border: Border.all(
+                color: Theme
+                  .of(context)
+                  .primaryColor,
+                width: 1.0,
+              ),
+              boxShadow: [
+                BoxShadow(
+                  color: Theme
+                    .of(context)
+                    .primaryColor,
+                  offset: Offset.zero,
+                  blurRadius: 10.0,
+                )
+              ],
+            ),
+            child: Center(
+              child: Text(child.description),
+            )),
+        )
+    ];
+    final Widget flexWidget = properties.type == Row
+      ? Row(children: children)
+      : Column(children: children);
+    final String flexWidgetName = properties.type.toString();
+    return Dialog(
+      child: Container(
+        margin: const EdgeInsets.all(16.0),
+        child: Column(
+          children: [
+            Container(
+              margin: const EdgeInsets.only(bottom: 4.0),
+              child: Text(
+                'Story of the flex layout of your $flexWidgetName widget',
+                style: TextStyle(
+                  fontWeight: FontWeight.bold,
+                  fontSize: 20.0,
+                )),
+            ),
+            Expanded(
+              child: Container(
+                color: Theme
+                  .of(context)
+                  .primaryColor,
+                child: Container(
+                  margin: const EdgeInsets.fromLTRB(8.0, 8.0, 0.0, 0.0),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Text(
+                        flexWidgetName,
+                        style: inspector_text_styles.regularBold,
+                      ),
+                      Expanded(
+                        child: Container(
+                          margin: const EdgeInsets.all(16.0),
+                          child: flexWidget)),
+                    ],
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+@immutable
+class FlexProperties {
+  const FlexProperties({
+                         this.direction,
+                         this.mainAxisAlignment,
+                         this.mainAxisSize,
+                         this.crossAxisAlignment,
+                         this.textDirection,
+                         this.verticalDirection,
+                         this.textBaseline,
+                         this.size,
```

Comment:
this formatting looks java-ey to me. Can you indent the parameters 2 spaces to match other Dart constructors?

Reply:
make sure to run dartfmt on the code and whatever it does is fine with the caveat that we may ask you to add trailing commas to force dartfmt to use better flutter style.

Reply:
seems like I misconfigured my styling configuration in my IntelliJ, I found it weird too but since it's generated by dartfmt I thought it was legit.
Sorry for this!